[32mINFO    [0m Common.Http:Http.py:164 2022-12-09 18:24:01.694576 | 
<-----Create Safe Account------>
Url:http://54.251.78.181:8888/escrow/api//accounts/safe
Headers:{"Content-Type": "application/json", "Authorization": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIxNjAxMDQzOTMwMjg3NDExMjAyIiwiZXhwIjoxNzAyMDg5NjAwLCJpYXQiOjE2NzA1NTM2MDAsInVzZXJJZCI6MTYwMTA0MzkzMDI4NzQxMTIwMn0.fZVc267JNsgy5tB_Wn89l2Zgd3ZgXJfaMg_T9lc3j-o"}
Body:{"name": "safe", "networkCode": "MATIC", "owner": "0244fb46bba2e912f26a73126b89742ed7f521f593ee084953ae008172553a0113", "recovery": ""}


[31m[1mERROR   [0m Common.Http:Http.py:170 2022-12-09 18:24:01.694576 | 
<-----Create account Response Error----->
{"code":2300000,"message":"Connect to 172.31.19.149:8080 [/172.31.19.149] failed: Connection refused \u0028Connection refused\u0029 executing POST http://core-center/core/api/accounts"}